javascript-loops-practice/countdown.js

```diff
@@ -0,0 +1,8 @@
+/* exported countdown */
+function countdown(number) {
+  var array = [];
+  for (var i = number; i > -1; i--) {
```

Comment:
This is a great example of where a `while` loop would be useful. Since we don't know how big `number` will be, it makes the most sense to use `while`. Both methods are completely fine and correct, just wanted to offer an alternative solution!